### PR TITLE
Feature implement mailer

### PR DESCRIPTION
**Pull request by sandrely2611-zz**
@FerPerales @anymoto tests feature implement mailer notification

### CODE REVIEW

File: app/models/internal_post.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ class InternalPost < Post
     self.body_html = parser.render(body_markdown)
   end
 
-  after_create :notify_twitter
+  #after_create :notify_twitter
```
Comment (anymoto) [inline app/models/internal_post.rb]: This line shouldn't be commented.

**Comment by anymoto** [comment]
@sandramu26 @KarenVentura please remove test/mailers/comments_mailer_test.rb, it's not being used.

File: app/models/comment.rb [hunk 2]
```diff
@@ -4,4 +4,9 @@ class Comment < ActiveRecord::Base
 
   validates_presence_of :user
   validates_presence_of :parent
+
+  after_create :notification_mailer
+  def notification_mailer
+    CommentMailer.notification_new_comment(self.parent.user_id, self.parent_id).deliver!
```
Comment (sandrely2611-zz) [inline app/models/comment.rb]: this line is necessary, such as is. @anymoto @FerPerales  everything is alright :+1:

File: Gemfile.lock [hunk 3]
```diff
@@ -233,6 +233,12 @@ GEM
       powerpack (~> 0.1)
       rainbow (>= 1.99.1, < 3.0)
       ruby-progressbar (~> 1.4)
+    rubocop (0.32.1)
```
Comment (FerPerales) [inline Gemfile.lock]: Why rucocop is here but not in the Gemfile?

File: app/mailers/application_mailer.rb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "from@example.com"
```
Comment (FerPerales) [inline app/mailers/application_mailer.rb:2]: Can you please make this email an env variable?

File: app/mailers/comment_mailer.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+class CommentMailer < ApplicationMailer
+  default from: "powercodegirls@gmail.com"
```
Comment (FerPerales) [inline app/mailers/comment_mailer.rb]: Can you please make this email an env variable?

File: app/mailers/comment_mailer.rb [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+  def notification_new_comment(user_id, parent_id)
+    @user = User.find user_id
+    @parent_id = parent_id
+    @url = "https://www.rubygamedev.com/internal_posts/#{@parent_id}"
```
Comment (FerPerales) [inline app/mailers/comment_mailer.rb:8]: Same here: make this configurable according to the environment

File: app/views/comment_mailer/notification_new_comment.text.erb [hunk 7]
```diff
@@ -0,0 +1,9 @@
+Welcome to example.com, <%= @user.username %>
```
Comment (FerPerales) [inline app/views/comment_mailer/notification_new_comment.text.erb]: Please update this copy to match the one in the HTML version

File: config/environments/production.rb [hunk 8]
```diff
@@ -70,7 +70,7 @@
     :user_name => ENV["MANDRILL_USERNAME"],
     :password  => ENV["MANDRILL_API_KEY"]
   }
-  
+
```
Comment (FerPerales) [inline config/environments/production.rb:73]: Extra line

File: spec/mailers/comment_mailer_spec.rb [hunk 9]
```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+describe CommentMailer do
+  describe "#notification_new_comment" do
+    let!(:comment) { FactoryGirl.build(:comment) }
+    let(:mail) { CommentMailer.notification_new_comment
+      (comment.parent.user.id, nil) }
```
Comment (FerPerales) [inline spec/mailers/comment_mailer_spec.rb]: What is that nil for?

**Pull request closed without merging**